src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Organization.ps1

```diff
@@ -1846,7 +1846,7 @@ class Organization: ADOSVTBase
             $responseObj = @([WebRequestHelper]::InvokeGetWebRequest($apiURL));
 
             $guestAccounts =  @()
-            if(($null -ne $responseObj) -and $responseObj.Count -gt 0 -and ([Helpers]::CheckMember($responseObj[0], 'members')))
+            if($responseObj.Count -gt 0 -and ([Helpers]::CheckMember($responseObj[0], 'members')))
```

Comment:
What if responeObj is null? $responseObj.Count will be error?

Reply:
$responseObj will never come null due to @(); at line 1846